RecoPixelVertexing/PixelTrackFitting/python/PixelTracks_cff.py

```diff
@@ -80,7 +80,9 @@
 
 # Use Riemann fit and substitute previous Fitter producer with the Riemann one
 from Configuration.ProcessModifiers.riemannFit_cff import riemannFit
+from Configuration.ProcessModifiers.riemannFitGPU_cff import riemannFitGPU
 riemannFit.toModify(pixelTracks, Fitter = "pixelFitterByRiemannParaboloid")
+riemannFitGPU.toModify(pixelTracks, runOnGPU = True)
```

Comment:
To enable the GPU-Riemann on the gpu workflow, it would be sufficient to change this to
```python
(riemannFitGPU | gpu).toModify(...)
```